File: src/main/java/com/ptah/entity/organization/Organization.java
package com.ptah.entity.organization;

import com.ptah.entity.contract.ContractParty;
import com.ptah.entity.user.User;
import jakarta.persistence.Entity;
import jakarta.persistence.Inheritance;
import jakarta.persistence.InheritanceType;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Entity
public class Organization extends ContractParty {
    private String businessCode;
}

File: backend/src/main/kotlin/com/github/jarryzhou/ptah/entity/userprofiling/Organization.kt
package com.github.jarryzhou.ptah.entity.userprofiling

import com.github.jarryzhou.ptah.entity.contract.Participant
import jakarta.persistence.Entity
import jakarta.persistence.OneToOne

@Entity
class Organization : Participant() {
    var businessCode: String? = null

    @OneToOne
    var contactInfo: ContactInfo? = null
}
